packages/animations/lib/src/open_container.dart

```diff
@@ -329,6 +352,7 @@ class _OpenContainerRoute extends ModalRoute<void> {
         assert(closedBuilder != null),
         assert(hideableKey != null),
         assert(closedBuilderKey != null),
+        assert(transitionType != null),
         _elevationTween = Tween<double>(
           begin: closedElevation,
           end: openElevation,
```

Comment:
(can't comment on the right line, but) do we need to update the color tween?

Reply:
Done